packages/multi-trait-rubric/configure/src/index.js

```diff
@@ -16,22 +16,28 @@ export default class MultiTraitRubricElement extends HTMLElement {
     this._configuration = configurationWithDefaults();
   }
 
-  validateModel = (m) => {
-    const validatedModel = m;
+  updateModelAccordingToReceivedProps = (m) => {
+    if (!m) {
+      return { ...defaults.model };
+    }
+
+    const validatedModel = { ...m };
     const { scales, excludeZero } = validatedModel;
 
     (scales || []).forEach(scale => {
+      if (!scale) return;
```

Comment:
Let's make sure we actually update the model and make it valid, Returning undefined here will break the code.